terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -167,6 +179,22 @@ resource "kubernetes_cron_job" "workflow_manager" {
               args = [
                 "--k8s-namespace", var.kubernetes_namespace,
                 "--input-bucket", var.ingestion_bucket,
```

Comment:
I'm guessing `input-bucket` is used by `workflow-manager` but `ingestor-input` is what goes to `facilitator`? Seems fine, especially since this may shortly get replaced by a configmap.